dotcom-rendering/src/components/Masthead/HighlightsCard.tsx

```diff
@@ -100,6 +100,13 @@ const content = css`
 	${from.tablet} {
 		padding-bottom: 10px;
 	}
+
+	/* We're deliberately using a font-size that is not in Source at a particular breakpoint */
+	${between.mobileMedium.and.mobileLandscape} {
```

Comment:
Would it be worth elaborating on why we use this irregular font size? eg something like "so that the headline breaks onto the same number of lines across all mobile breakpoints"

Reply:
Good idea, I'll add the "why"